crates/sui-deepbook-indexer/src/migrations/2024-07-17-121456_deepbook/up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS progress_store
+(
+    task_name                   TEXT         PRIMARY KEY,
+    checkpoint                  BIGINT       NOT NULL,
+    target_checkpoint           BIGINT       NOT NULL,
+    timestamp                   BIGINT       NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS deepbook
```

Comment:
this is where you come and change schema, then I usually run `diesel migration redo --database-url <db_url>` and everything is ready on the postgres side